javascript/core/intro-js/javascript-basics-part-1.md

```diff
@@ -1,22 +1,10 @@
 ---
 author: alexjmackey
 
-levels:
-  - beginner
-
 type: normal
 
 category: must-know
 
-inAlgoPool: false
-
-standards:
-  javascript.javascript-syntax.1: 10
-  javascript.javascript-syntax.2: 10
-
-tags:
-  - introduction
-
 aspects:
   - introduction
 
```

Comment:
Line 18

Since this is the same sentence as in `javascript-basic-part-2`
Shouldn't this one also be changed to:

`Like most programming languages, JavaScript has a number of syntax rules that it must adhere to.`

Reply:
I tried tagging line 18 but couldn't